lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -388,7 +388,7 @@ defmodule DynamicSupervisor do
           Supervisor.child_spec()
           | {module, term}
           | module
-          | (old_erlang_child_spec :: :supervisor.child_spec())
+          | :supervisor.child_spec()
```

Comment:
```suggestion
          | (old_erlang_child_spec :: :supervisor.child_spec())
```

You were too quick. :D